<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuCard</class>
 <widget class="QWidget" name="menuCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none;</string>
  </property>
  <widget class="QFrame" name="mainContainer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>501</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"> background: #FFBA71; border-radius: 5;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="dayLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white; color: black; border: none;</string>
       </property>
       <property name="text">
        <string>ПОНЕДЕЛЬНИК</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="menuFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QFrame" name="bottomFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>351</width>
          <height>141</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>140</horstretch>
          <verstretch>140</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>140</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>351</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QPushButton" name="updateButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none; background: white; color: black;</string>
            </property>
            <property name="text">
             <string>UPD</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="favoriteButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none; background: white; color: black;</string>
            </property>
            <property name="text">
             <string>ИЗБР</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <widget class="QFrame" name="rightSide">
            <property name="styleSheet">
             <string notr="true">background: white; color: black; border: none;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>241</width>
               <height>111</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="paramsVerticalLayout"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QFrame" name="productList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>351</width>
          <height>241</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none; background: white; color: black;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>331</width>
           <height>221</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="menuVerticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="menuTitle_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string>МЕНЮ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
